interface/globals.php

```diff
@@ -191,6 +191,10 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Variable set for Eligibility Verification [EDI-271] path
 $GLOBALS['edi_271_file_path'] = $GLOBALS['OE_SITE_DIR'] . "/edi/";
 
+//  Necessary writeable paths for mPDF tool
+define("_MPDF_TEMP_PATH", $GLOBALS['OE_SITE_DIR'].'/documents/mpdf/pdf_tmp/');
+define("_MPDF_TTFONTDATAPATH", $GLOBALS['OE_SITE_DIR'].'/documents/mpdf/ttfontdata/');
```

Comment:
Complete and committed. I sure miss using shorthand! I have not done any testing on Linux so hoping you are testing there. Thanks for the input.